services/navigation/builtin/builtin.go

```diff
@@ -354,13 +347,19 @@ func (svc *builtIn) Reconfigure(ctx context.Context, deps resource.Dependencies,
 		return err
 	}
 
+	// Create explore motion service
+	exploreMotionConf := resource.Config{ConvertedAttributes: &explore.Config{}}
```

Comment:
[note] Will add error here before merging to keep explore mode turned off until after realworld testing has occurred

Reply:
should this error still be added?